cardano-api/src/Cardano/Api/Utils.hs

```diff
@@ -59,6 +61,9 @@ Just x  ?! _ = Right x
 Left  e ?!. f = Left (f e)
 Right x ?!. _ = Right x
 
+throwNothingM :: Monad m => e -> Maybe a -> ExceptT e m a
```

Comment:
Thoughts on the name `throwOnNothingM`? Also why the M?

Reply:
I think the `&` reads okay.  It's a well known operator that I just read it as "and".

For example `f & throwOnNothing` I would read as do `f` and throw on nothing.

I think keeping the `&` is also better because it fits in with other operators like `<&>` that we might want to use in other contexts.